CHANGELOG.md

```diff
@@ -16,6 +16,13 @@ All notable changes to this project will be documented in this file.
 
 - Adeunis - ftdNetworkTester added default schema & uplink test
 
+# [1.1.1] - 2021-09-23
```

Comment:
this should now be the newest version

Reply:
will change the other versions then if this gets approved

Reply:
you should add it as the newest version, instead of adding a new version in between